core/embed/trezorhal/unix/display_driver.c

```diff
@@ -27,46 +27,9 @@
 #include "common.h"
 #include "profile.h"
 
-#define EMULATOR_BORDER 16
-
-#if defined TREZOR_MODEL_T
-
-#ifdef TREZOR_EMULATOR_RASPI
-#define WINDOW_WIDTH 480
-#define WINDOW_HEIGHT 320
-#define TOUCH_OFFSET_X 110
-#define TOUCH_OFFSET_Y 40
-#else
-#define WINDOW_WIDTH 400
-#define WINDOW_HEIGHT 600
-#define TOUCH_OFFSET_X 80
-#define TOUCH_OFFSET_Y 110
-#endif
-
-#elif defined TREZOR_MODEL_1
-
-#define WINDOW_WIDTH 200
-#define WINDOW_HEIGHT 340
-#define TOUCH_OFFSET_X 36
-#define TOUCH_OFFSET_Y 92
-
-#elif defined TREZOR_MODEL_R
-
-#define WINDOW_WIDTH 193
-#define WINDOW_HEIGHT 339
-#define TOUCH_OFFSET_X 32
-#define TOUCH_OFFSET_Y 84
-
-#elif defined TREZOR_MODEL_T3T1
-
-#define WINDOW_WIDTH 400
-#define WINDOW_HEIGHT 600
-#define TOUCH_OFFSET_X 80
-#define TOUCH_OFFSET_Y 110
+#include TREZOR_BOARD
```

Comment:
`TREZOR_BOARD` is already included in `xdisplay.h`. However, if you prefer to keep it here, I would suggest placing it at the beginning as the first included file. Although we do not have formal rules regarding this...

Reply:
moved https://github.com/trezor/trezor-firmware/commit/4aeb7aa83bfc4969221049687ea8839140ee20be